static.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "application": {
+    "apiHost": "http://localhost:4001/dev",
+    "env": "development",
+    "version": "development",
+    "defaultResponseHeaders": {
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Allow-Headers": "*",
+      "Access-Control-Allow-Credentials": true
+    }
+  }
+}
```

Comment:
this line does not have a trailing new line. I suggest you create an .editorconfig file and define the rules. https://editorconfig.org